args/args_test.go

```diff
@@ -32,13 +31,15 @@ func TestSetupArgsDefault(t *testing.T) {
 		Verbose: false, Pretty: false, Hostname: "localhost", Port: 3306,
 		Username: "", Password: "",
 	}
+
+	// default is overridden
+	args.Hostname = "localhost"
```

Comment:
hm... I don't understand this. `localhost` is a default value. Why default is overridden and we need to set it again?